viewing_party/party.py

```diff
@@ -1,23 +1,201 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    if not title or not genre or not rating:
+        return None
+    movie_dict = {
+        'title': title,
+        'genre': genre,
+        'rating': rating
+    }
+    return movie_dict

```

Comment:
If we are creating a variable/data structure and then immediately returning it, feel free to get into the habit of just returning the dictionary! We don't need the variable as a man in the middle!

```suggestion
    return {
        'title': title,
        'genre': genre,
        'rating': rating
    }
    

```